dev-docs/adunit-reference.md

```diff
@@ -0,0 +1,342 @@
+| `aspect_ratios` | Optional | Array[Object]                         | Instead of `sizes`, you can define allowed aspect ratios.  For properties, see [`image.aspect_ratios`](#native-image-aspectratios). |
+
+<a name="native-image-aspectratios" />
+
+###### `adUnit.mediaTypes.native.image.aspect_ratios`
+
+{: .table .table-bordered .table-striped }
+| Name           | Scope    | Type    | Description                                                                                          |
+|----------------+----------+---------+------------------------------------------------------------------------------------------------------|
+| `min_width`    | Optional | Integer | The minimum width required for an image to serve (in pixels).                                        |
+| `ratio_height` | Required | Integer | This, combined with `ratio_width`, determines the required aspect ratio for an image that can serve. |
+| `ratio_width`  | Required | Integer | See above.                                                                                           |
+
+<a name="native-title" />
+
+##### `adUnit.mediaTypes.native.title`
+
+{: .table .table-bordered .table-striped }
+| Name       | Scope    | Type    | Description                                          |
+|------------+----------+---------+------------------------------------------------------|
+| `required` | Required | Boolean | Whether a title asset is required on this native ad. |
+| `len`      | Optional | Integer | Length of title text, in characters.                 |
+
+<a name="native-sponsoredBy" />
+
+##### `adUnit.mediaTypes.native.sponsoredBy`
+
+{: .table .table-bordered .table-striped }
+| Name       | Scope    | Type    | Description                                               |
+|------------+----------+---------+-----------------------------------------------------------|
+| `required` | Required | Boolean | Whether a brand name asset is required on this native ad. |
+
+<a name="native-clickUrl" />
+
+##### `adUnit.mediaTypes.native.clickUrl`
+
+{: .table .table-bordered .table-striped }
+| Name       | Scope    | Type    | Description                                              |
+|------------+----------+---------+----------------------------------------------------------|
+| `required` | Required | Boolean | Whether a click URL asset is required on this native ad. |
+
+##### `adUnit.mediaTypes.native.body`
+
+{: .table .table-bordered .table-striped }
+| Name       | Scope    | Type    | Description                                       |
+|------------+----------+---------+---------------------------------------------------|
+| `required` | Required | Boolean | Whether body text is required for this native ad. |
+
+<a name="native-icon" />
+
+##### `adUnit.mediaTypes.native.icon`
```

Comment:
If there's a clean way to consolidate native.icon with native.image, we should do so, since the properties below are identical.

Reply:
I share that desire, but I don't think we can implement it in a way that isn't a little confusing.  Right now the page has specific anchor links to object properties, which will be handy when linking from other pages, e.g.,

`<a name="adUnit.mediaTypes.native.icon" />`

If we consolidate I'm not sure how to make clear to someone clicking in from another page, who wants `adUnit.mediaTypes.native.icon.aspect_ratios` but gets the image version, that "oh these are the image docs but they're the same for icon".  We could add notes, except that we have to duplicate the note in a couple places so it's not much of a win dupe-wise IMO.